callback/error_nashi.js

```diff
@@ -0,0 +1,27 @@
+const { Database } = require("sqlite3").verbose();
+const db = new Database(":memory:");
+
+const createTable = `
+  CREATE TABLE books(
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    title TEXT NOT NULL UNIQUE
+  )
+`;
+
+db.run(createTable, (error) => {
+  db.run(
+    "INSERT INTO books(title) VALUES(?)",
+    ["これを読めば非同期処理がわかる本"],
+    function (error) {
+      console.log(this.lastID);
+
+      db.get("SELECT * FROM books WHERE id = ?", [1], (error, row) => {
```

Comment:
`this.lastID` を使って取得を行ったほうが良い気がします。

```suggestion
      db.get("SELECT * FROM books WHERE id = ?", [this.lastID], (error, row) => {
```

Reply:
動的に指定したほうが良いですね。修正しました:pray: